xenserver/types.go

```diff
@@ -23,7 +23,7 @@ import (
 	"log"
 	"strconv"
 
-	"github.com/ringods/go-xen-api-client"
+	"github.com/gingods/go-xen-api-client"
```

Comment:
The url 'github.com/gingods/go-xen-api-client' doesn't exist. Is this a typo ?

Reply:
@mborodin can you revert this typo?

Reply:
fixed